addons/website/models/ir_ui_view.py

```diff
@@ -469,6 +475,8 @@ def _set_noupdate(self):
     def save(self, value, xpath=None):
         self.ensure_one()
         current_website = self.env['website'].get_current_website()
+        if current_website:
+            self = self.with_context(_view_website_saved=True)
```

Comment:
why this `_view_website_saved`?

Reply:
To differentiate a backend updated field from one updated through website builder - and therefore not track changes done in backend. But if the feature becomes "track every use of web_editor", this makes no more sense indeed.

Reply:
Removed the need for such a context variable.